src/digest/digest.cr

```diff
@@ -1,12 +1,18 @@
 require "base64"
 
-class Digest::FinalizedError < Exception
-end
+# The Digest module contains implementations of hashing algorithms like
```

Comment:
This doc comment could be more elaborate and refer to `Digest` as the base type for implementations (instead of an container).